### DISCUSSION TITLE
How to enable search with filename

### DISCUSSION THREAD

**Discussion by vinay-keshava** [jameslittle230/stork#348] [Q&A]
How to enable search with filename ?

**Comment by jameslittle230** [comment]
Hi @vinay-keshava - I don't believe that the feature that you're describing exists in Stork today.

Are you describing a feature where you'd be able to specify a document name and a search term and find the location of that search term only within the given file?

Thanks,
James

**Reply by vinay-keshava** [reply]
>Are you describing a feature where you'd be able to specify a document name and a search term and find the location of that search term only within the given file?

yes

**Comment by jameslittle230** [comment]
Also - moving this over to Discussions for now
